week-1/H-numbers/npm-debug.log

```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```

Comment:
This file shouldn't have been committed. Be careful you don't accidentally add the wrong files when running `git add`.